src/main/kotlin/app/api/SourcererApi.kt

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.api
+
+import app.model.Commit
+import app.model.Repo
+import app.model.User
+
+/**
+ * Sourcerer API.
+ */
+object SourcererApi : Api {
+    var api: Api = ServerApi()
```

Comment:
Actually, SourcererAPI is such API user it is also pervasive because API used across app.